mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -676,13 +697,16 @@
         <Alignment value="Negative"/>
         <InfoIcon value="Assets\Textures\btn-upgrade-terran-interferencematrix.dds"/>
         <EditorCategories value="AbilityorEffectType:Units"/>
-        <Duration value="15"/>
+        <Duration value="11"/>
```

Comment:
Raven Interference Matrix duration reduced from 11 to 8 seconds (from 15 to 11 on normal speed)

The rounding of the real seconds is off in the patch notes, but whatever...